nlp/2021_06_29_long_range_transformers/LongRangeTransformers.ipynb

```diff
@@ -0,0 +1,1940 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "7XEv8E2SALpN"
+   },
+   "source": [
+    "# 🏉 Go long! Long range transformers\n",
+    "\n",
+    "In recent research there has been an extensive study for improving the calculation of attention in transformer architectures. Mostly for improving their capacity to handle longer token sequences. 👊\n",
+    "\n",
+    "The attention calculation is known to be quadratic in compatation time with respect to the sequence length 👎. These recent advances, however, are able to perform attention calculation in linear time with respect to the sequence length. This allows us to scale the transformer architecture such that it can handle input sequences beyond the usual token length of 512. \n",
+    "\n",
+    "In this notebook, we compare traditional transformers with novel efficient transformers. We'll use roBERTa as a baseline to compare against LongFormer and BigBird.  \n",
+    "\n",
+    "Let's put these architectures to the test and see which one comes out on top 🏆!  \n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "l8ymZTD_BSZS"
+   },
+   "source": [
+    "## 🛠️ Getting started: Install packages & download models\n",
```

Comment:
well technically we're not downloading any models here

Reply:
Indeed 🙈